### DISCUSSION TITLE
railway要强制付费了，有考虑提供其他云函数部署方式吗

### DISCUSSION THREAD

**Discussion by JMPTW0** [zdz/ServerStatus-Rust#239] [General]
> The Developer plan will be discontinued on August 1, 2023. Switch to the Hobby plan to continue using Railway.
以上

**Comment by zdz** [comment]
docker 部署的方式都是类似的，可以试试 fly.io / render.com ， fly.io  绑个卡，每个月有5刀的免费额度，部署这个 server 也够用了

**Comment by zdz** [comment]
railway 平替 https://app.koyeb.com/  ，配置类似，就是地区目前除了FRA 区，其它区要自己手动申请

![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/c0cca396-6d4a-4e62-8c78-7fa914e5f464)
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/7661fec6-c91a-4ec8-99fd-0e64839c6aa4)
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/3a4523c4-40da-4105-a8ff-14d6f5b63e76)
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/97497e88-8804-406f-927b-8a893f473d31)

偶尔会部署失败，可以重试部署
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/2f16703c-a0d5-42d9-ae0c-2a2b663250b5)

**Reply by lithromantic** [reply]
SRV_CONF 不生效怎么办，应该是分行的问题，设置里面重新设置也不行
